### PR TITLE
Issue 510: Remove empty text from group events list

### PR DESCRIPTION
**Pull request by kasperg**
This prevents the view from appearing if there is no events for the group and we can allow news to use the full width.

Issue: http://platform.dandigbib.org/issues/510

This is necessary for ding2/ddbasic#6 to work.

### CODE REVIEW

File: ding_event.info [hunk 1]
```diff
@@ -95,3 +95,4 @@ features[variable][] = scheduler_publish_enable_ding_event
 features[variable][] = scheduler_publish_touch_ding_event
 features[variable][] = scheduler_unpublish_enable_ding_event
 features[views_view][] = ding_event
+mtime = 1427142274
```
Comment (cableman) [inline ding_event.info]: Please remove the mtime from the info file

**Comment by cableman** [comment]
Please update commit message to match standard :-)

**Comment by kasperg** [comment]
@cableman: Done.

**Comment by cableman** [comment]
Reviewed and approved

**Pull request merged**